package.json

```diff
@@ -85,8 +85,7 @@
     "release_note:generate": "scripts/use_node scripts/generate_release_note",
     "cypress:run-without-security": "env TZ=America/Los_Angeles NO_COLOR=1 cypress run --headless --env SECURITY_ENABLED=false",
     "cypress:run-with-security": "env TZ=America/Los_Angeles NO_COLOR=1 cypress run --headless --env SECURITY_ENABLED=true,openSearchUrl=https://localhost:9200,WAIT_FOR_LOADER_BUFFER_MS=500",
-    "osd:ciGroup10": "echo \"dashboard_sanity_test_spec.js\"",
-    "osd:ciGroup11": "echo \"apps/vis_builder/*.js\"",
```

Comment:
removed `"osd:ciGroup11": "echo \"apps/vis_builder/*.js\"",` because vis_builder tests are already running in ciGroup1 as shown here https://github.com/opensearch-project/opensearch-dashboards-functional-test/blob/main/package.json#L28